_data/master/navbars/getting-started.yml

```diff
@@ -33,6 +33,8 @@ toc:
       path: /getting-started/kubernetes/installation/vagrant/
   - title: Tutorials
     section:
+    - title: Using calicoctl
```

Comment:
I put it here because it is kubernetes specific, and "Usage" is for orchestrator independent calico docs.